### PR TITLE
Pa01

### PR DESCRIPTION
**Pull request by KevinMartinLankut**
Jeg Kevin Martin Lankut og Kaspar Lilja gjorde ferdig pa01.

### CODE REVIEW

**Review by Kasparlilja — APPROVED** [review]

File: src/YOUR_NAME_ex/pa01/snakes_and_ladders.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+"""
+Source:
+https://codereview.stackexchange.com/questions/176586/snakes-and-ladders-game
+"""
+
+import random
+import statistics
+
+
+def move_player(player, current_pos):
```
Comment (danielambrosius) [inline src/YOUR_NAME_ex/pa01/snakes_and_ladders.py:15]: why is `player` an argument to this function? It doesn't look like its used...

File: src/YOUR_NAME_ex/pa01/snakes_and_ladders.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        players[player] = 0
+
+    # Start game
+    start = 1
+    moves = 0
+    while not start == 0:
+        for player, current_pos in players.items():
+
+            # Total moves
+            moves += 1
```
Comment (danielambrosius) [inline src/YOUR_NAME_ex/pa01/snakes_and_ladders.py:40]: it looks like you're increasing `moves` for each player and not for each round of the game. You will then end up with `n` times more moves then the winning number for `n` players. Line 40 should be moved just above line 37

File: src/YOUR_NAME_ex/pa01/snakes_and_ladders.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    return next_pos
+
+
+def single_game(num_players):
+    players = {}
+    for player in range(1, num_players + 1):
+        players[player] = 0
+
+    # Start game
+    start = 1
```
Comment (danielambrosius) [inline src/YOUR_NAME_ex/pa01/snakes_and_ladders.py:34]: this start value is a bit odd. I would call it something like `won = False`. That is a bit more explicit.

**Review by danielambrosius — APPROVED** [review]
Approved. BUT: There are no docstrings. In your next pair assignment i expect to see docstrings for all the important functions. If not i will fail it.

**Pull request merged**